sdk/src/reserved_account_keys.rs

```diff
@@ -162,6 +167,7 @@ lazy_static! {
         ReservedAccount::new_active(stake::program::id()),
         ReservedAccount::new_active(system_program::id()),
         ReservedAccount::new_active(vote::program::id()),
+        ReservedAccount::new_pending(zk_elgamal_proof_program::id(), feature_set::add_new_reserved_account_keys::id()),
```

Comment:
This should have used the `zk_elgamal_proof_program_enabled` feature id. By using `add_new_reserved_account_keys`, you've now pushed back the minimum version that we can enable the `add_new_reserved_account_keys` feature from v2.0 to v2.1

Reply:
Maybe I'm missing something, but this PR was added before 2.0 was branched, so it's part of 2.0, along with the `add_new_reserved_account_keys`. So they are part of the same release and can be enabled in the normal cadence.

You're right that it would have made more sense to use `zk_elgamal_proof_program_enabled` here though. With `add_new_reserved_account_keys`, the zk elgamal proof program key will be a reserved key before the program is enabled, which is a bit strange. But I don't see why it would push back enabling `add_new_reserved_account_keys` to v2.1

Reply:
Argh, apologies, I spelled Elgamal wrong when grepping git history. So this PR is fine as is, sorry for the noise. I do think it would have more sense to use the `zk_elgamal_proof_program_enabled` feature anyways but it's not a big deal in my opinion to reserve the id before it's enabled and so not worth switching at this point.